packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -82,14 +88,127 @@ const List = ({
   );
 };
 
+/**
+ * Generic vertically-centered Header layout
+ */
+const Header = ({
+  flexBoxProps,
+  headingProps,
+  heading,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  heading: string;
+  flexBoxProps?: ComponentProps<typeof FlexBox>;
+  headingProps?: ComponentProps<typeof Heading>;
+}) => (
+  <Slide {...rest}>
+    <FlexBox height="100%" {...flexBoxProps}>
+      <Heading {...headingProps}>{heading}</Heading>
+    </FlexBox>
+  </Slide>
+);
+
+/**
+ * Section layout with left aligned text
+ */
+const Section = ({
+  sectionTitle,
+  sectionTitleProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  sectionTitle: string;
+  sectionTitleProps?: ComponentProps<typeof Heading>;
+}) => (
+  <Header
+    heading={sectionTitle}
+    headingProps={sectionTitleProps}
+    flexBoxProps={{ justifyContent: 'flex-start' }}
+  />
+);
+
+/**
+ * Statement layout with centered text
+ */
+const Statement = ({
+  statement,
+  statementProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  statement: string;
+  statementProps?: ComponentProps<typeof Heading>;
+}) => <Header heading={statement} headingProps={statementProps} />;
+
+/**
+ * Big Fact with optional fact information
+ */
+const BigFact = ({
+  fact,
+  factInformation,
+  factProps,
+  factInformationProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  fact: string;
+  factInformation?: string;
+  factProps?: ComponentProps<typeof Text>;
+  factInformationProps?: ComponentProps<typeof Text>;
+}) => (
+  <Slide {...rest}>
+    <FlexBox>
+      <Box>
+        <Text fontSize="250px" textAlign="center" {...factProps}>
+          {fact}
+        </Text>
+        {factInformation ? (
+          <Text textAlign="center" {...factInformationProps}>
+            {factInformation}
+          </Text>
+        ) : null}
+      </Box>
+    </FlexBox>
+  </Slide>
+);
+
+/**
+ * Quote layout
+ */
+const Quote = ({
+  quote,
+  quoteProps,
+  attribution,
+  attributionProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  quote: string;
+  quoteProps?: ComponentProps<typeof Text>;
+  attribution: string;
+  attributionProps?: ComponentProps<typeof Text>;
+}) => (
+  <Slide {...rest}>
+    <Box width="100%" margin="auto">
+      <Text fontSize="85px" {...quoteProps}>
+        {quote}
+      </Text>
+      <Text fontSize="36px" padding={'0em 0em 0em 1em'} {...attributionProps}>
+        {attribution}
```

Comment:
```suggestion
        &ndash; {attribution}
```

What are your thoughts on adding an `ndash` here?

Reply:
I like it! 👍